### ISSUE TITLE
Heroic Launcher's new reload feature hijacking CTRL+R in the background. 

### ISSUE THREAD

**Issue by GarnetSunset** [Heroic-Games-Launcher/HeroicGamesLauncher#2692]
### Describe the bug

When heroic launcher is open in the tray, heroic launcher hijacks the CTRL+R reloads I attempt to send to firefox, and the CTRL+SHIFT+I I send to Discord. When closing heroic from the taskbar, CTRL+R and CTRL+SHIFT+I work in other places again. Please fix! Really annoying and I was worried I had a virus haha.

### Add logs

```text
(11:06:52) INFO:    [Legendary]:        Refreshing library...
(11:06:52) INFO:    [Legendary]:        Refreshing Epic Games...
(11:06:52) INFO:    [Connection]:       Connectivity: check-online
(11:06:52) INFO:    [Connection]:       Pinging external endpoints
(11:06:52) INFO:    [Legendary]:        Running command: C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary --version
(11:06:52) INFO:    [Legendary]:        Legendary location: C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary
(11:06:52) INFO:    [Gog]:              GOGDL location: C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl
(11:06:52) INFO:    [Gog]:              GOGDL location: C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl
(11:06:52) INFO:    [Legendary]:        Game list updated, got 178 games & DLCs
(11:06:52) WARNING: [Backend]:          Protocol already registered.
(11:06:53) INFO:    [Connection]:       Connectivity: online
(11:06:53) INFO:    [Gog]:              Checking if login is valid
(11:06:53) INFO:    [Gog]:              Running command: C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl --auth-config-path C:\Users\James\AppData\Roaming\heroic\gog_store\auth.json auth
(11:06:53) INFO:    [Legendary]:        Running command: C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary list --third-party
(11:06:53) INFO:    [Gog]:              Running command: C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl --auth-config-path C:\Users\James\AppData\Roaming\heroic\gog_store\auth.json --version
(11:06:53) INFO:    [Backend]:          Loading Screen Ready
(11:06:53) INFO:    [Frontend]:         Refreshing Library
(11:06:53) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(11:06:53) INFO:    [Gog]:              Found 0 game(s) to update
(11:06:53) INFO:    [Backend]:          Frontend Ready
(11:06:53) INFO:    [Backend]:          Checking for new Heroic Updates
(11:06:53) INFO:    [Backend]:          Checking for current version changelog
(11:06:53) INFO:    [Legendary]:        Checking for game updates: C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary list --third-party
(11:06:53) ERROR:   [Backend]:          Error when checking for current Heroic changelog
(11:06:53) ERROR:   [Backend]:          Error when checking for Heroic updates Error: Request failed with status code 403
    at createError (C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar\node_modules\axios\lib\core\createError.js:16:15)
    at settle (C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar\node_modules\axios\lib\core\settle.js:17:12)
    at IncomingMessage.handleStreamEnd (C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar\node_modules\axios\lib\adapters\http.js:322:11)
    at IncomingMessage.emit (node:events:525:35)
    at endReadableNT (node:internal/streams/readable:1359:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
(11:06:53) INFO:    [Gog]:              Saved username to config file
(11:06:53) INFO:    [Backend]:          VCRuntime is installed
(11:06:54) INFO:    [Legendary]:        Found 0 games to update
(11:06:55) INFO:    [Backend]:          

Heroic Version: 2.7.1 Eustass Kid
Legendary Version:  0.20.32 Dark Energy (hotfix #6)
GOGdl Version: 0.7.1

Electron Version: 23.1.4
Chrome Version: 110.0.5481.192
NodeJS Version: 18.12.1

OS: Microsoft Windows 11 Pro KERNEL: 10.0.22621 ARCH: x64
CPU: AMD Ryzen 9 7950X3D 16-Core Processor @4.2 
RAM: Total: 63.15 GiB Available: 45.08 GiB
GRAPHICS: GPU0: NVIDIA GeForce RTX 4080 VRAM: 16376MB DRIVER: 531.79GPU1: Microsoft Basic Display Adapter  


(11:06:56) INFO:    [Frontend]:         Refreshing Library
(11:06:56) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(11:06:56) INFO:    [Gog]:              Found 0 game(s) to update
(11:06:56) INFO:    [Backend]:          Checking for new Heroic Updates
(11:06:56) ERROR:   [Backend]:          Error when checking for Heroic updates Error: Request failed with status code 403
    at createError (C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar\node_modules\axios\lib\core\createError.js:16:15)
    at settle (C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar\node_modules\axios\lib\core\settle.js:17:12)
    at IncomingMessage.handleStreamEnd (C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar\node_modules\axios\lib\adapters\http.js:322:11)
    at IncomingMessage.emit (node:events:525:35)
    at endReadableNT (node:internal/streams/readable:1359:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
(11:06:56) INFO:    [Backend]:          Checking for current version changelog
(11:06:56) ERROR:   [Backend]:          Error when checking for current Heroic changelog
(11:06:56) INFO:    [Legendary]:        Checking for game updates: C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary list --third-party
(11:06:57) INFO:    [Legendary]:        Found 0 games to update
(11:06:57) INFO:    [Frontend]:         Refreshing Library
(11:06:57) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(11:06:57) INFO:    [Gog]:              Found 0 game(s) to update
(11:06:57) INFO:    [Legendary]:        Checking for game updates: C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary list --third-party
(11:06:57) INFO:    [Backend]:          Checking for new Heroic Updates
(11:06:57) ERROR:   [Backend]:          Error when checking for Heroic updates Error: Request failed with status code 403
    at createError (C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar\node_modules\axios\lib\core\createError.js:16:15)
    at settle (C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar\node_modules\axios\lib\core\settle.js:17:12)
    at IncomingMessage.handleStreamEnd (C:\Users\James\AppData\Local\Programs\heroic\resources\app.asar\node_modules\axios\lib\adapters\http.js:322:11)
    at IncomingMessage.emit (node:events:525:35)
    at endReadableNT (node:internal/streams/readable:1359:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
(11:06:57) INFO:    [Backend]:          Checking for current version changelog
(11:06:57) ERROR:   [Backend]:          Error when checking for current Heroic changelog
(11:06:57) INFO:    [Legendary]:        Found 0 games to update
(11:06:58) INFO:    [Backend]:          Starting the Download Queue
```

### Steps to reproduce

1. Open Heroic
2. Minimize to taskbar
3. Try to reload a page with CTRL+R in (assumedly) any browser
4. Does not happen in the browser of choice. 

### Expected behavior

It should *NOT* hijack CTRL+R etc. 

### Screenshots

https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/13430434/7913195c-464b-4d09-99fb-a78ca9890fbf

### Heroic Version

Latest Stable

### System Information

- Windows 11

**Comment by joshandersn** [comment]
Awesome! i was just about to say i have this same problem in macOS

**Comment by miquelfire** [comment]
Only today I found it was Heroic's fault when I just decided to just close it as a debugging step. For me, it was Ctrl+K that stopped working. I use that to create links in Google Docs, but with how I configured my Firefox, it's also a shortcut to the search bar (when you don't have a single box for URL and search). Also, Ctrl+L was taken by Heroic, though Ctrl+Shift+L isn't (password manager shortcut for me).

**Comment by arielj** [comment]
the hijacked shortcuts are: ctrl+r ctrl+q ctrl+j ctrl+k ctrl+l and ctrl+shift+i

next release will include the fix

**Comment by GarnetSunset** [comment]
For those of you still dealing with this I recommend grabbing a build from the buildbot 

https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/actions/runs/4968167820

**Comment by arielj** [comment]
closing, this should be fixed now in 2.8.0
